libs/agora/gene-comparison-tool/src/lib/gene-comparison-tool.component.ts

```diff
@@ -172,27 +181,32 @@ export class GeneComparisonToolComponent implements OnInit, AfterViewInit, OnDes
   @ViewChild('scorePanel') scorePanel!: ScorePanelComponent;
   @ViewChild('pinnedGenesModal') pinnedGenesModal!: PinnedGenesModalComponent;
 
-  constructor() {
-    this.category = 'RNA - Differential Expression';
-  }
-
   ngOnInit() {
-    this.urlParamsSubscription = this.route.queryParams.subscribe((params) => {
-      this.urlParams = params || {};
+    this.category = this.categoryParam ? this.categoryParam : this.categories[0].value;
```

Comment:
What do you think about creating a `defaultCategory` variable that can be used here and to initialize `category` on line 110?

```suggestion
    this.category = this.categoryParam ? this.categoryParam : this.defaultCategory;
```

Reply:
In this case, the category varaiable is already initialized